server/mailet/integration-testing/src/test/java/org/apache/james/transport/mailets/AmqpForwardAttachmentTest.java

```diff
@@ -200,10 +179,8 @@ public void stripAttachmentShouldPutAttachmentsInMailAttributeWhenConfiguredForI
             calmlyAwait.atMost(Duration.ONE_MINUTE).until(messageSender::messageHasBeenSent);
             calmlyAwait.atMost(Duration.ONE_MINUTE).until(() -> imapMessageReader.userReceivedMessage(RECIPIENT, PASSWORD));
         }
-        
-        boolean autoAck = true;
-        GetResponse basicGet = channel.basicGet(queueName, autoAck);
-        assertThat(basicGet.getBody()).isEqualTo(TEST_ATTACHMENT_CONTENT);
+
+        assertThat(amqpRule.readContentAsBytes()).contains(TEST_ATTACHMENT_CONTENT);
```

Comment:
amqpRule.readContentAsBytes() returns an optional

This semantic is as powerfull as what was returns previously, because we check the string contained in the optional is equal to, and not that the string contains